src/Game.elm

```diff
@@ -87,14 +87,15 @@ type alias PlayingState =
     { cueBallPosition : Point3d Meters WorldCoordinates
     , cueElevation : Angle
     , hitElevation : Angle
-    , hitRelativeAzimuth : Angle
-    , shootButton : Maybe Duration
-    , mouse : PlayingMouse
+    , hitRelativeAzimuth : Angle -- offset from camera azimuth
+    , shootButton : Maybe Posix -- the time when the button was pressed
     }
 
 
-type PlayingMouse
-    = HoveringCueBall -- TODO: use to render the clickable area
+type
+    PlayingMouse
+    -- TODO: use HoveringCueBall to render the clickable area
+    = HoveringCueBall Angle Angle -- elevation and relative azimuth
```

Comment:
Since these are the same type, would it make it clearer to use a record?
```
type PlayingMouse
    = HoveringCueBall HoveringData

type alias HoveringData =
    { elevation : Angle
    , azimuth : Angle
    }
```